zipkin-storage/elasticsearch/src/main/java/zipkin2/elasticsearch/ElasticsearchVersion.java

```diff
@@ -39,6 +39,8 @@ float get(HttpCall.Factory callFactory) throws IOException {
     try {
       if (enterPath(parser, "version", "number") != null) version = parser.getText();
     } catch (RuntimeException | IOException possiblyParseException) {
+      throw new IllegalArgumentException(
```

Comment:
On phone so can't expand this file to see below but if it's dumping the next line, comme t instead is fine